besu/src/main/java/org/hyperledger/besu/Besu.java

```diff
@@ -62,7 +62,13 @@ public static void main(final String... args) {
   }
 
   private static Logger setupLogging() {
-    InternalLoggerFactory.setDefaultFactory(Log4J2LoggerFactory.INSTANCE);
+    try {
+      InternalLoggerFactory.setDefaultFactory(Log4J2LoggerFactory.INSTANCE);
+    } catch (Throwable t) {
+      System.out.printf(
+          "Could not set netty log4j logger factory: %s - %s%n",
+          t.getClass().getSimpleName(), t.getMessage());
```

Comment:
If you didn't do this would it continue running besu without error?

Also wondering why we're using a netty factory here and not slf4j/log4j?

Reply:
I'm not sure why we configure log4j with netty, it's probably some netty magic.

The reason to wrap it in an exception is that I'm setting things up so log4j can be removed at the classpath level and things continue to function.  (a) in case there is another log4shell and (b) in case someone uses the out-of-band slf4j configurations or a wrapper class to point the logging at java.util.logging, logback, google logging, or elsewhere, and then wants to remove unneeded classes